USER_GUIDE.md

```diff
@@ -124,4 +124,72 @@ func main() {
     fmt.Println("deleting index", deleteIndexResponse)
 }
 
-```
\ No newline at end of file
+```
+
+## How to use IAMs as authentication method
+
+Before starting, we strongly recommend to read the full AWS documentation regarding IAMs for Opensearch.
```

Comment:
"Before starting, we strongly recommend reading the full AWS documentation regarding using IAM credentials to sign to requests to OpenSearch APIs."